configure.ac

```diff
@@ -1,14 +1,14 @@
 #
-# Copyright (c) 2019, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# Copyright (c) 2019-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 #
 # See file LICENSE for terms.
 #
 AC_PREREQ([2.63])
 
 AC_COPYRIGHT([Copyright (c) 2019, NVIDIA CORPORATION & AFFILIATES. All rights reserved.])
 
-define([nccl_rdma_sharp_plugins_ver_major], 1)
-define([nccl_rdma_sharp_plugins_ver_minor], 0)
+define([nccl_rdma_sharp_plugins_ver_major], 2)
+define([nccl_rdma_sharp_plugins_ver_minor], 1)
```

Comment:
should we set minor number to 3,  as we are currently building 2.3 branch?

Reply:
Maybe change to `2.4` for `master` and `2.3` for `v2.3.x` branch (in separate PR)?

Reply:
Updated to 2.4